en/news/_posts/2017-09-18-ruby-2-5-0-preview1-released.md

```diff
@@ -0,0 +1,52 @@
+---
+layout: news_post
+title: "Ruby 2.5.0-preview1 Released"
+author: "naruse"
+translator:
+date: 2017-09-18 00:00:00 +0000
+lang: en
+---
+
+We are pleased to announce the release of Ruby 2.5.0-preview1.
+
+Ruby 2.5.0-preview1 is the first preview release toward Ruby 2.5.0.
+It introduces some new features and performance improvements, for example:
+
+## New Features
+
+* Print backtrace and error message in reverse order if STDERR is unchanged and a tty.
+  [Feature #8661] [experimental]
+
+* Top-level constant look-up is removed.  [Feature #11547]
+
+* rescue/else/ensure are allowed inside do/end blocks.  [Feature #12906]
+
+* yield\_self [Feature #6721]
+
+## Other notable changes since 2.3
+
+* Merge Onigmo to 6.1.1.
+  It adds [absent operator](https://github.com/k-takata/Onigmo/issues/82)
```

Comment:
Upstream changed the name of the operator to "absence operator" at https://github.com/k-takata/Onigmo/issues/87#issuecomment-291848558